test/bats/run-cr.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+
+load test_helper
+
+CR_SRC=yamaszone/clusterrunner:src
+CR_DEMO_REPO=https://github.com/boxengservices/ClusterRunnerDemo.git
+
+@test "'run-cr' script displays help." {
+	run ./run-cr
+	assert_contains "$output" "Usage:"
+}
+
+@test "CR-BIN: Help msg displayed by image built from binary." {
+	run ./run-cr bin -h
+	assert_contains "$output" "usage: clusterrunner"
```

Comment:
@josephharrington I have created a dedicated repo for ClusterRunner Docker image [here](https://github.com/yamaszone/clusterrunner-docker-image/compare/develop?expand=1). Please let me know your thoughts...